src/app/components/filter/filter.component.ts

```diff
@@ -17,8 +20,8 @@ export class FilterComponent implements OnInit {
 
   isShow: boolean = false;
   searchForm: FormGroup;
-  // TODO: skills and cities are currently hardcoded but need to be retrieved from the backend at some point
-  skills: string[] = ['Angular', 'HTML', 'Java', 'JUnit', 'Mockito', 'Postgres', 'Spring'];
+  skills: string[] = ['Java', 'Spring', 'Angular', 'HTML', 'Postgres', 'Mockito', 'JUnit'];
+  vacancies: IVacancies[] = [];
```

Comment:
Duplicate line. Lines 24 and 26 are the same. One of them should be removed as the application won't work correctly.

Reply:
adapted